src/utils/types/yup/shema.ts

```diff
@@ -0,0 +1,60 @@
+import * as yup from 'yup';
+
+export const SCHEMA = yup.object().shape({
+  name: yup
+    .string()
+    .matches(/^[A-Z].*$/, '⚠️ First letter should be uppercase')
+    .required('⚠️ Name is required'),
+  age: yup
+    .number()
+    .transform((value) => {
```

Comment:
Your typescript and eslint configuration require to specify the parameter type explicitly and it shouldn't be "any"

Reply:
fixed